Changes.md

```diff
@@ -21,6 +21,7 @@ Improvements
 - OpenImageIOReader : The `availableFrames` plug no longer errors for a missing file sequence - instead it outputs an empty list.
 - PathListingWidget : Header data is now computed asynchronously, without locking the UI.
 - CollectScenes : Improved cancellation responsiveness for large lists of `rootNames`.
+- EditScopeUI : Added the ability to turn off visibility of selected objects in the viewer using the <kbd>Ctrl</kbd>+<kbd>H</kbd> shortcut.
```

Comment:
This is in the section for the already-released 1.2.9.0, rather than the yet-to-be-released 1.2.x.x.

Reply:
Yep, little fella snuck in there without me noticing. Fixed in https://github.com/GafferHQ/gaffer/pull/5381/commits/833593820d6fc2cca4882c3e0d1784a640437c4d and https://github.com/GafferHQ/gaffer/pull/5381/commits/00188e8bb234c254769e30bc631c57a5781c8502